### DISCUSSION TITLE
Last.fm scrobbler screen to link the account does not appear.

### DISCUSSION THREAD

**Discussion by Haizeaa1** [pear-devs/pear-desktop#2640] [Q&A]
I've turned on the scrobbler including the last.fm plugin and restarted the app and have had no screen appear for me to link the account on any occassion

**Comment by JellyBrick** [comment]
Enable last.fm in the scrobbler plugin setting

**Reply by Haizeaa1** [reply]
I have done that as well, the screen still doesn't appear 
![image](https://github.com/user-attachments/assets/3b0de7e4-d0dc-4de4-adba-adf7ec8ad98d)

**Comment by thlost0001** [comment]
i have the same problem. idk what to do

**Comment by froll27** [comment]
I had the same issue, What fixed it is that I downloaded the RPM version instead of the flatpak one and restarted the computer and it worked.
